app/src/main/res/values/strings.xml

```diff
@@ -279,6 +279,10 @@
     <string name="first_aid_guide_prompt">Basic first aid training tips and procedures for any emergency !</string>
     <string name="guide_search_bar_prompt">In case of emergency, you can quickly find what you\'re looking for thanks to our available search bar! </string>
     <string name="KEY_USER_AGREE">USER_AGREE</string>
+    <string name="KEY_USER_SIGNED_IN">USER_SIGNED_IN</string>
+    <string name="KEY_USER_UID">USER_UID</string>
+    <string name="KEY_USER_NAME">USER_NAME</string>
```

Comment:
If these are only used in the code, it is maybe better to define them as public constants in one of the files they are being used. This is to avoid having to use to getString(id) method to get them

Reply:
It was defined like this by the teammates who implemented the local storage and it makes more sense to keep them like that because they are keys of the local storage, and thus can be accessed from anywhere in the app